.gitignore

```diff
@@ -17,5 +17,6 @@
 *.class
 *.pyc
 *.html
+.idea/
```

Comment:
Rather simply delete the .idea files for now and in another PR you can do this 😉